docs/api/actions.md

```diff
@@ -104,6 +104,45 @@ function readPlanets () {
 repo.push(readPlanets)
 ```
 
+### Return a generator
+
+Often times we need to dispatch multiple actions in sequential
+order. For example, what if we want to ask the user to confirm their
+action before deleting a record?
+
+This can be accomplished by using a generator:
+
+```javascript
+function ask (message) {
+  return action => {
+    if (confirm(message)) {
+      action.resolve()
+    } else {
+      action.reject()
+    }
+  }
+}
+
+function deleteUser (id) {
+  return fetch.delete('/users/${id}').then(response => response.json())
+}
+
+function confirmAndDelete (user) {
+  return function * (send) {
```

Comment:
I wonder if it would be a little less cryptic to have this function accept a `repo` instance, and the yield calls would be:
```js
yield repo.push(ask, `blabalbla`)
yield repo.push(deleteUser, user.id)
```